### ISSUE TITLE
mock-input-debug-panel not building

### ISSUE THREAD

**Issue by slaymaker1907** [2graphic/sinap-ide#164]
It is giving a similar error as we were seeing yester

```
    ERROR in C:\Users\gagnier\Documents\GitHub\2graphic\sinap-ide\node_modules\sinap-core\lib\src\types.d.ts
    (2,29): error TS2307: Cannot find module '../sinap-includes/types-interfaces'.
```

**Comment by RighteousRaichu** [comment]
It's the same issue as yesterday, just copy the file where it needs to be in the mean time

**Comment by slaymaker1907** [comment]
In order to fix this as suggested by @RighteousRaichu, copy /node_modules/sinap-core/sinap-includes/types-interfaces.d.ts to /node_modules/sinap-core/lib/sinap-includes.

**Comment by Sheyne** [comment]
Check if it's fixed in `0.3.1`
